.github/workflows/main-branch.yml

```diff
@@ -24,6 +24,9 @@ jobs:
     name: Python Cross Platform Build & Test
     uses: ./.github/workflows/python-cross-platform-support.yml
 
-  build_and_deploy_documentation:
-    name: Build and Deploy GitHub Pages
-    uses: ./.github/workflows/sphinx.yml
+# Commenting out from main branch workflow till permissions aspect can be resolved for 
```

Comment:
Believe my added comment explains the drive for this change. Also, will let us evaluate if anything else within the main workflow has been impacted by the repo change and permissions needing updated.

Reply:
I'm not tracking what the underlying issue for the page permission/token issue, are you familiar with that issue or is this something we need to invest some time into figuring out?

Reply:
My initial assessment thinks it's just likely to be the pages sourcing might not have been properly configured. But I can't confirm without access to settings for the repo. 
https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site

Seems to be problematic for automated publishing, but manual "bypasses".

Reply:
I see. That's so odd...

Reply:
Yeah, the write ups on pages and their sources are a bit confusing on that portion. Single contributions (which I think of as manual) can be contributed to in varied ways with seeming no issue. But routine contributions (so like via the pipeline), has to have repo level configuration in place correctly.